services/api/src/game/__tests__/game-room.service.test.ts

```diff
@@ -20,22 +26,24 @@ describe('gameRoomService', () => {
     expect(secondCall).toMatchSnapshot();
   });
 
-  it('should cleanup room', async () => {
+  it('should print rooms', async () => {
     gameRoomService.getOrCreateRoom(roomId);
-    gameRoomService.cleanUp(roomId);
-    expect(gameRoomService.getClients(roomId)).toBeUndefined();
+    gameRoomService.printRooms();
+    expect(logger.debug).toBeCalled();
   });
 
-  it.each([roomId, 'does-not-exist'])(
-    'should get turns undefined',
-    async (id) => {
-      expect(gameRoomService.getTurns(id)).toBeUndefined();
-    },
-  );
+  it('should not clean up room, because there are less than 1000', () => {
+    gameRoomService.getOrCreateRoom(roomId);
+    gameRoomService.cleanUp();
+    expect(gameRoomService.getClients(roomId)).toBeDefined();
+  });
 
-  it('should print rooms', async () => {
+  it('should clean up the room, because there are more than 1000', () => {
```

Comment:
update test to be independent on other tests.

Reply:
```
  it('should clean up the room, because there are more than 1000', () => {
    for (let i = 0; i < 1001; i++) {
      gameRoomService.getOrCreateRoom(`${i}`);
    }
    expect(gameRoomService.rooms.size).toBe(1001);
    gameRoomService.cleanUp();
    expect(gameRoomService.rooms.size).toBe(1000);
  });```